sections/image-with-text.liquid

```diff
@@ -50,7 +50,7 @@
     <div class="image-with-text__text-item grid__item">
       <div
         id="ImageWithText--{{ section.id }}"
-        class="image-with-text__content image-with-text__content--{{ section.settings.desktop_content_position }} image-with-text__content--desktop-{{ section.settings.desktop_content_alignment }} image-with-text__content--mobile-{{ section.settings.mobile_content_alignment }} image-with-text__content--{{ section.settings.height }} gradient color-{{ section.settings.color_scheme }} content-container"
+        class="image-with-text__content image-with-text__content--{{ section.settings.desktop_content_position }} image-with-text__content--desktop-{{ section.settings.desktop_content_alignment }} image-with-text__content--mobile-{{ section.settings.mobile_content_alignment }} image-with-text__content--{{ section.settings.height }} gradient color-{{ section.settings.color_scheme }} content-container{% if settings.animations_reveal_on_scroll and section.settings.content_layout == 'overlap' %} scroll-trigger animate--slide-in{% endif %}"
```

Comment:
Is there an element below this wrapper that we can place the animation on? Right now, when you use a background colour, the colour fades in as well and it looks a bit off. In previous PRs, @kjellr asked to make sure that the background for different elements doesn't fade in, but the individual elements do.

**Original**

https://user-images.githubusercontent.com/22390758/234898759-3c75e22f-965f-4d0e-877b-ba66cf7a08cf.mp4

**Slow motion**


https://user-images.githubusercontent.com/22390758/234898873-cd9d5bba-de8e-4de6-9aba-ff7163962203.mp4

Reply:
It doesn't bother me too much in this case — the rule about the background color seems less important to me when it's only taking up half of the screen like this.

Reply:
Thank you for putting words on what originally bothered me, but I assume it was expected because of the transitions. I am curious to see how this could experience without fade!

Reply:
Here is how it looks, ill push the change so you can test. 

There was something else I needed to fix in the logic. Make sure to test when there isn't any overlap too :) 

[Video example](https://screenshot.click/27-41-92inc-twnjv.mp4)

Reply:
Yeah, I think the fade is preferable. Let's use that!